src/styles/GlobalStyle.js

```diff
@@ -0,0 +1,43 @@
+import { createGlobalStyle } from 'styled-components';
+import { brandFonts } from '../styles';
+import sofiaProBoldTTF from '../fonts/sofia-pro-bold.ttf';
+import sofiaProRegTTF from '../fonts/sofia-pro-regular.ttf';
+
+const GlobalStyle = createGlobalStyle`
+    @font-face {
+        font-family: 'Sofia Pro';
+            src: url('${sofiaProBoldTTF}') format('ttf'),
```

Comment:
This need to be truetype as the format not ttf. Fonts don't seem to render correctly. Also need to add Sofia Pro Regular and Sofia Pro Bold as all fonts are currently coming through as bold or find a different way to add both but being able to use font weights as well